wscript

```diff
@@ -243,6 +247,13 @@ def _build_dynamic_sources(bld):
         ],
     )
 
+    bld(
+        features='uavcangen',
```

Comment:
we can clean it up after this PR I think. At this stage I'd like to get PR in as I think it is a big step forward from current master